dev-util/rt-tests/rt-tests-1.10.ebuild

```diff
@@ -32,7 +32,7 @@ src_compile() {
 }
 
 src_install() {
-	emake prefix=/usr DESTDIR="${D}" MAN_COMPRESSION=none install
+	emake prefix=/usr DESTDIR="${D}" install
```

Comment:
I think we need ${EPREFIX} _or_ ${ED} here.

Reply:
Yeah. It seems like that should be `emake prefix="${EPREFIX}"/usr DESTDIR="${D}" install`.